packages/infra/lib/api-stack/api-service.ts

```diff
@@ -246,6 +251,11 @@ export function createAPIService(
           ],
           resources: ["*"],
         }),
+        new iam.PolicyStatement({
+          actions: ["geo:SearchPlaceIndexForText"],
+          resources: [`arn:aws:geo:*`], // Perhaps, need to be more specific with the exact resource here
+          effect: iam.Effect.ALLOW,
+        }),
```

Comment:
Can't we use `placeIndex` to grant access? Cool if we can't, I also didn't find an easy way to do that for `apigateway:GET`, but better if we can.

Reply:
I was referring to use something like `placeIndex.grantAccess(fargateService.taskDefinition.taskRole);`, if there's something like this on AWS Place service's API.

Reply:
ya, there wasn't anything like that, so I went for what we have now... 😒